go/vt/sqlparser/ast.go

```diff
@@ -2560,6 +2639,60 @@ type (
 	JSONUnquoteExpr struct {
 		JSONValue Expr
 	}
+
+	// ArgumentLessWindowExpr stands for the following window_functions: CUME_DIST, DENSE_RANK, PERCENT_RANK, RANK, ROW_NUMBER
+	ArgumentLessWindowExpr struct {
+		Type       ArgumentLessWindowExprType
+		OverClause *OverClause
+	}
```

Comment:
Is this struct for window expression which don't take an argument?

Reply:
Yes